portfolio/src/main/webapp/galleryStyle.css

```diff
@@ -0,0 +1,56 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+img {
+  vertical-align: middle;
+  width: inherit;
+}
+
+.slideshow {
+  margin: auto;
+  position: relative;
+  width: 750px;
+}
+
+.EuropeSlideshow {
```

Comment:
nit: usually proper nouns do not get special treatment when it comes to naming in code (i.e. `europeSlideshow` would be consistent)

Reply:
Got it! Changing that now.